### ISSUE TITLE
Changing the length of the vector changes the results of annual glare

### ISSUE THREAD

**Issue by mostaphaRoudsari** [mostaphaRoudsari/Honeybee#243]
This is super strange! I need to look into this in details and if I can't figure it out just remove it for now. Here is another question that I sent before and I got no good reply. I wish there was better documentation for Daysim.

![image](https://cloud.githubusercontent.com/assets/2915573/5795885/91dbbbfc-9f68-11e4-95c6-a5918b2e03ca.png)

**Comment by antonszilasi** [comment]
@mostaphaRoudsari  is his the vector of the camera?

**Comment by mostaphaRoudsari** [comment]
@antonszilasi yes however this shouldn't be an issue with honeybee as it does normalize the vector for the analysis.
